boards/arm/efr32_slwstk6061a/doc/efr32_slwstk6061a.rst

```diff
@@ -145,6 +145,17 @@ Connect the EFR32-SLWSTK6061A to your host computer using the USB port and you
 should see a USB Serial Port. Use `J-Link`_ or Silicon Labs Simplicity Studio
 to flash the generated zephyr.bin.
 
+It is also possible to flash with `OpenOCD tool`_, however the version of
+OpenOCD available in Zephyr SDK is too old and does not support EFR32FG1P chip,
```

Comment:
Let's clarify and break into two sentences, with this change:

    OpenOCD included in the Zephyr SDK v0.9.3 is too old and
    does not support the EFR32FG1P chip.  You will need to ...